src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -242,7 +242,15 @@ const constructPageFetchUrl = ({
         break;
       case HOME_PAGE: {
         const variantPath = variant ? `/${variant}` : '';
-        fetchUrl = Url(`/${service}${variantPath}/${id}`);
+
+        if (isLocal) {
+          fetchUrl = Url(`/${service}${variantPath}`);
+        } else {
+          const homePageId = service
+            ? HOME_PAGE_CONFIG?.[service]?.[env]
+            : null;
+          fetchUrl = Url(`/${service}${variantPath}/${homePageId ?? ''}`);
+        }
```

Comment:
Is this change needed? We're in the `isLocal` condition for this switch statement, so I'm quite sure what this is doing.

Reply:
I've updated it as you're right, I doubled up on the `isLocal` as I was getting an odd scoping issue with `jest` when trying to fix the tests but the latest change doesn't need it and simplifies the code so hopefully we're all good, thanks